src/main/java/co/inventrosoft/springboottask/service/TournamentService.java

```diff
@@ -0,0 +1,95 @@
+package co.inventrosoft.springboottask.service;
+
+import co.inventrosoft.springboottask.console.ConsoleParser;
+import co.inventrosoft.springboottask.console.MatchResult;
+import co.inventrosoft.springboottask.model.Match;
+import co.inventrosoft.springboottask.model.Team;
+import co.inventrosoft.springboottask.repository.TournamentRepositoryJdbcImpl;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+@Service
+@AllArgsConstructor
+public class TournamentService {
+    private final MatchService matchService;
+    private final TeamService teamService;
```

Comment:
DB services and Repositories should not meet together.
Possible cyclic dependency here.